### DISCUSSION TITLE
Remove peakAnnotation/deviations matrix

### DISCUSSION THREAD

**Discussion by jindalk** [GreenleafLab/ArchR#1347] [General]
Hi, is it possible to remove a custom peakAnnotation and the corresponding deviations matrix from an ArchR project?

**Comment by rcorces** [comment]
I guess it depends why you are trying to do this. You could obviously overwrite the matrix in question. The peakAnnotations are actually stored as separate files in the "Annotations" sub-directory of your project. So those can be manually deleted and you could set the corresponding entries in `ArchRProj@peakAnnotation` to `NULL`.

In terms of removing a matrix, see this post. not perfect but its currently the only way
https://github.com/GreenleafLab/ArchR/discussions/843

**Reply by jindalk** [reply]
Got it, thank you!
